quiz/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+services:
+  backend_quiz:
+    build:
+        context: ./backend
+    command: python manage.py runserver 0.0.0.0:8000
+    env_file:
+      - ./backend/.envs/.env
+      - ./.envs/.env_db
+    ports:
+      - 8000:8000
+    depends_on:
+      - backend_db
+    restart: on-failure
+    volumes:
+      - ./backend:/app
+    networks:
+      - backend_network
```

Comment:
`backend_network` wygląda na sieć która nic nie łączy.

Reply:
jak tą sieć wywalam, to nginx nie znajduje mi hosta dla backendu

Reply:
Teraz jest spoko wcześniej wisiała sama 😃